src/utils.py

```diff
@@ -99,3 +110,37 @@ def print_os_environment():
     """
     for key, val in os.environ.items():
         print(f"{key}: {val}")
+
+
+def get_semver_command(body):
+    match = re.match(
+        rf".*\B/(?P<{COMMAND}>{COMMAND_MAJOR}|{COMMAND_MINOR}|{COMMAND_PATCH})\b",
+        body,
+        re.IGNORECASE,
+    )
+    if match:
+        return match.group(COMMAND).lower()
+
+
+def semver_label_in_pull(pull):
+    return any(
```

Comment:
```python
return list(filter(lambda x: x in get_labels(pull=pull), labels))
```

Reply:
why filter here? it does not make the code any clearer.